server/modules/statisticalFactsheets/api.js

```diff
@@ -0,0 +1,113 @@
+const Manager = require('./manager')
+const VersionService = require('../../versioning/service')
+
+module.exports.init = (app) => {
+  app.get('/statisticalFactsheets/', async (req, res) => {
+    const schemaName = await VersionService.getDatabaseSchema()
+    const [
+      extentOfForest,
+      growingStockTotal,
+      carbonStock,
+      forestCharacteristics,
+      forestAreaWithinProtectedAreas,
+      specificForestCategories,
+      forestOwnership,
+      primaryDesignatedManagementObjective,
+    ] = await Promise.all([
+      Manager.getExtentOfForest(schemaName),
+      Manager.getGrowingStockTotal(schemaName),
+      Manager.getCarbonStock(schemaName),
+      Manager.getForestCharacteristics(schemaName),
+      Manager.getForestAreaWithinProtectedAreas(schemaName),
+      Manager.getSpecificForestCategories(schemaName),
+      Manager.getForestOwnership(schemaName),
+      Manager.getPrimaryDesignatedManagementObjective(schemaName),
+    ])
+
+    res.json({
+      carbonStock,
+      extentOfForest,
+      forestAreaWithinProtectedAreas,
+      forestCharacteristics,
+      forestOwnership,
+      growingStockTotal,
+      primaryDesignatedManagementObjective,
+      specificForestCategories,
+    })
+  })
+
+  app.get('/statisticalFactsheets/forestCharacteristics', async (req, res) => {
+    const schemaName = await VersionService.getDatabaseSchema()
+    const forestCharacteristics = await Manager.getForestCharacteristics(schemaName)
+
+    res.json({
```

Comment:
do you need to return a named object with one property only?

Reply:
Yes, I use this value in PowerBI